scripts/verify.js

```diff
@@ -0,0 +1,106 @@
+//web3 modules
+const Web3 = require('web3');
+
+//general purpose npm moudles
+const fs = require('fs');
+const assert = require('assert');
+const compareVersions = require('compare-versions');
+const solc = require('solc');
+
+process.on('unhandledRejection', console.error.bind(console))
+
+//contract sources
+const controllerContractPath = "./../contracts/controller/";
+const factoryContractPath = "./../contracts/factory/";
+const tokenContractPath = "./../contracts/token/";
+const utilsContractPath = "./../contracts/utils/";
+
+const compilationInput = {
+    "OwnableContract.sol" : fs.readFileSync(utilsContractPath + 'OwnableContract.sol', 'ascii'),
+    "OwnableContractOwner.sol" : fs.readFileSync(utilsContractPath + 'OwnableContractOwner.sol', 'ascii'),
+    "IndexedMapping.sol" : fs.readFileSync(utilsContractPath + 'IndexedMapping.sol', 'ascii'),
+    "Controller.sol" : fs.readFileSync(controllerContractPath + 'Controller.sol', 'ascii'),
+    "ControllerInterface.sol" : fs.readFileSync(controllerContractPath + 'ControllerInterface.sol', 'ascii'),
+    "Factory.sol" : fs.readFileSync(factoryContractPath + 'Factory.sol', 'ascii'),
+    "Members.sol" : fs.readFileSync(factoryContractPath + 'Members.sol', 'ascii'),
+    "MembersInterface.sol" : fs.readFileSync(factoryContractPath + 'MembersInterface.sol', 'ascii'),
+    "WBTC.sol" : fs.readFileSync(tokenContractPath + 'WBTC.sol', 'ascii')
+};
+
+function findImports (_path) {
+    if(_path.includes("openzeppelin-solidity"))
+        return { contents: fs.readFileSync("./../node_modules/" + _path, 'ascii') }
+    else
+        return { contents: fs.readFileSync("./../contracts/" + _path, 'ascii') }
+}
+
+const mainnetUrls = ['https://mainnet.infura.io',
+                     'https://api.mycryptoapi.com/eth'];
+
+
+const wbtcAddress = "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599";
+
+checkAll();
+
+async function checkAll() {
+  await(check(mainnetUrls[0]));
+  await(check(mainnetUrls[1]));
```

Comment:
why runt it twice? it results in duplicated outputs. suggest to run in on one of the endpoints and keep the other one written in the file as backup.

Reply:
ugly, if single endpoint fails can run it with the other one.